_templates/exchanges.html

```diff
@@ -56,6 +56,7 @@ <h3 id="australia"><img src="/img/flags/AU.png" alt="Australian flag">Australia<
         <a href="https://www.cointree.com.au/">CoinTree</a><br>
         <a href="https://www.hardblock.net/">HardBlock</a><br>
         <a href="https://www.independentreserve.com/">Independent Reserve</a>
```

Comment:
A line break needs to be added here, similarly to the preceding entries, since Independent Reserve will no longer be the last entry under the Australian exchanges.